client/src/lib.rs

```diff
@@ -36,6 +40,16 @@ pub mod samples {
             transaction_add_nonce: DEFAULT_TRANSACTION_NONCE,
         }
     }
+
+    /// Gets network status seen from the peer in json format
+    ///
+    /// # Errors
+    /// Fails if sending request or decoding fails
+    pub fn get_status_json(client: &Client) -> Result<Status> {
+        let req = client.prepare_status_request::<DefaultRequestBuilder>();
+        let resp = req.build()?.send()?;
+        StatusResponseHandler::handle(&resp)
+    }
```

Comment:
Not sure if this method is needed at all in `iroha_client`..

Reply:
I think it should be fine as it's only used for the testing purposes in the designated module.